develop/blockchains/get-started/intro-polkadot-sdk.md

```diff
@@ -0,0 +1,86 @@
+
+- Networking and peer-to-peer communication (powered by [Libp2p](/polkadot-protocol/glossary#libp2p))
+- Consensus protocols, such as [BABE](/polkadot-protocol/glossary#blind-assignment-of-blockchain-extension-babe), [GRANDPA](/polkadot-protocol/glossary#grandpa), or [Aura](/polkadot-protocol/glossary#authority-round-aura)
+- Cryptography
+- A selection of pre-built modules, called [pallets](/polkadot-protocol/glossary#pallet)
+- Benchmarking and testing suites
+- The ability to create portable Wasm runtimes
+
+!!!note
+    For an in-depth dive into the monorepo, the [Polkadot SDK Rust documentation](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/index.html){target=\_blank} is highly recommended.
+
+## Polkadot SDK Overview
+
+The Polkadot SDK is composed of five major components:
+
+![](/images/develop/blockchains/get-started/intro-polkadot-sdk/polkadot-sdk-structure.webp)
+
+- [Substrate](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/substrate/index.html){target=\_blank} - a set of libraries for building blockchains
+- [FRAME](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/frame_runtime/index.html){target=\_blank} - a blockchain development framework built on top of Substrate
+- [Cumulus](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/cumulus/index.html){target=\_blank} - a set of libraries and pallets to add parachain capabilities to a Substrate/FRAME runtime
+- [XCM (Cross Consensus Messaging)](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/xcm/index.html){target=\_blank} - the primary format for conveying messages between parachains 
+- [Polkadot](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/polkadot/index.html){target=\_blank} - the node implementation for the Polkadot protocol
+
+### Substrate
+
+Substrate is a Software Development Kit (SDK) that uses Rust-based libraries and tools to enable you to build application-specific blockchains from modular and extensible components. Application-specific blockchains built with Substrate can run as standalone services or in parallel with other chains to take advantage of the shared security provided by the Polkadot ecosystem. Substrate includes default implementations of the core components of the blockchain infrastructure to allow you to focus on the application logic.
+
+Every blockchain platform relies on a decentralized network of computers—called nodes—that communicate with each other about transactions and blocks.
+In general, a node in this context is the software running on the connected devices rather than the physical or virtual machine in the network.
+As software, Substrate-based nodes consist of two main parts with separate responsibilities:
```

Comment:
We should create a diagram similar to the following for this section:
<img width="751" alt="image" src="https://github.com/user-attachments/assets/9553d519-afc9-4981-b54e-b8c1c6ccd7c6">

Reply:
I don't like how this one looks at all.  If you want to get this on a list of diagrams for after launch, please open a ticket to request a mermaid diagram, paste the text including the information that goes in the diagram (what are we trying to explain visually?), let me know which page it's for, and assign to me. If we find we can make some before launch then we will but I don't think these should keep us from shipping.